web/src/beta/components/PublishStateSwitchField/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { FC, useState } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+type Props = {
+  list: string[];
+  selected?: string;
+  onChange?: (label: string) => void;
```

Comment:
```suggestion
  labels: string[];
  selected: string;
  onChange: (label: string) => void;
```

- `list` as plural and `label` as singular might be a bit difficult to understand.
- `selected` looks like a boolean, but actually, it's string so might be better to change
- (IMO) No one might not use this component without `selected` or `onChange`, making these fields required might be better.